company-profiles/intuit.md

```diff
@@ -0,0 +1,46 @@
+# Intuit Inc.
+
+## Company blurb
+
+Intuit is a global technology platform with a mission to power prosperity around the world. Our innovative products like TurboTax, QuickBooks, Mint, Credit Karma and Mailchimp help millions of consumer, small business and self-employed customers overcome their most important financial challenges.
+
+Our products and services are built on Intuit's AI-driven expert platform, combining the power of cutting edge technology with live financial experts. Our team innovates together in a diverse and collaborative environment where technology makes a global impact. From uncovering data patterns for how wealth is created, to designing conversational interfaces that deliver personalized experiences, we’re putting AI, ML, and an unrivaled set of data to work to help millions of people achieve financial freedom.
+
+Our culture and entrepreneurial spirit have earned us a spot on “Best Companies to Work For” lists year after year. We’re dedicated to living our mission of powering prosperity, not just through our products, but through our social impact initiatives. Through our Prosperity Hubs, we’re bringing new jobs, teaching new skills, and supporting small businesses in distressed communities to help local economies rebuild and thrive.
+
+## Company size
+
+10,000+
+
+## Remote status
+
+We include these past jobs as a way for you to explore what kinds of remote and flexible jobs Intuit has hired for in the past and might be likely to hire for again in the future.
+[https://jobs.intuit.com/location/remote-jobs/27595/1000000000100/2](https://jobs.intuit.com/location/remote-jobs/27595/1000000000100/2)
+
+## Region
+Worldwide
+
+## Company technologies
```

Comment:
This section is used as a way for applicants to see what the technologies the company uses so that they know they have the skills before applying.

Could you list technologies that the company uses, for example Python, Nodejs, Docker etc